docs/how-to-guides/tracing/trace_span.md

```diff
@@ -0,0 +1,150 @@
+# Overview
+
+The span, as you may know, is the fundamental unit of the trace system, representing a unit that captures execution information in the PromptFlow system. Spans are nested together in a parent-child relationship and paired together by link relationships, providing developers and users with a comprehensive view of the application’s execution process.
+
+This document outlines the design of PromptFlow spans, detailing what information is traced and how it is structured. By adhering to these specifications, we ensure transparency and consistency in our tracing system.
```

Comment:
list the objectives user will learn or scenarios they can achieve after read this doc

Reply:
This document will serve as reference rather than a turtorial.